### DISCUSSION TITLE
最大化后还原的问题

### DISCUSSION THREAD

**Discussion by louiejordan** [electerm/electerm#3105] [General]
一直以来我都有一个疑问，不知道是不是系统的问题，我的系统是Windows11，软件右上角最大化之后再点击没反应，正常不都是还原的吗？还有就是双击也不能实现最大化和还原功能，是只有Windows这样还是所有系统都这样？

**Comment by zxdong262** [comment]
确实有问题，可以设置里开启系统标题栏，使用系统标题栏就没有这些问题了

**Reply by louiejordan** [reply]
系统标题栏有点丑
